notebook-cleanup/1-RuntimeScanning/xray-query.json

```diff
@@ -0,0 +1,6 @@
+{
+    "filters": {
+        "min_severity": "Critical",
+        "watch_name": "watch-k8s-acr"

```

Comment:
Will this watch build up over time with a large number of items in the list or is it flushed out periodically? Alternatively is there a time component to the filter that could be used if necessary to limit to recent scans?

Reply:
fair play, yeah we could use a `created_from` [field](https://www.jfrog.com/confluence/display/JFROG/Xray+REST+API#XrayRESTAPI-GetViolations) to ensure that we only grab 'new' violations that are generated with a specific pull